src/status_im2/contexts/communities/overview/view.cljs

```diff
@@ -219,142 +150,198 @@
               (i18n/label :t/joined)
               (i18n/label :t/pending))}])
 
-(defn render-page-content
+(defn calc-icon-top
+  [scroll-height]
+  (if (<= scroll-height scroll-page/negative-scroll-position-0)
+    -40
+    (->> (+ scroll-page/scroll-position-0 scroll-height)
+         (* (if platform/ios? 3 1))
+         (+ -40)
+         (min 8))))
+
+(defn calc-icon-size
+  [scroll-height]
+  (->> (+ scroll-page/scroll-position-0 scroll-height)
+       (* (if platform/ios? 3 1))
+       (- scroll-page/max-image-size)
+       (max scroll-page/min-image-size)
+       (min scroll-page/max-image-size)))
+
+(defn round-community-icon
+  [community scroll-height]
+  [rn/view
+   {:style (style/round-community-icon-style (calc-icon-top scroll-height))}
+   [quo/community-icon community
+    (calc-icon-size scroll-height)]])
+
+(defn community-token-gating-details
+  [name thumbnail-image tokens]
+  [rn/view {:height 200 :margin-top 20}
+   [quo/token-gating
+    {:community {:name                     name
+                 :community-color          colors/primary-50
+                 :community-avatar-img-src thumbnail-image
+                 :gates                    tokens}}]])
+
+(defn add-on-press-handler
+  [community-id {:keys [name emoji id locked? token-gating] :or {locked? false} :as chat}]
+  (merge
+   chat
+   (if (and locked? token-gating)
+     {:on-press #(open-channel-token-gating-details
+                  name
+                  token-gating
+                  emoji
+                  (colors/custom-color :pink 50))}
+
+     (when (and (not locked?) id)
+       {:on-press (fn []
+                    (rf/dispatch [:dismiss-keyboard])
+                    (rf/dispatch [:chat.ui/navigate-to-chat-nav2 (str community-id id)])
+                    (rf/dispatch [:search/home-filter-changed nil]))}))))
+
+(defn add-on-press-handler-to-chats
+  [community-id chats]
+  (mapv (partial add-on-press-handler community-id) chats))
+
+(defn add-on-press-handler-to-categorized-chats
+  [community-id categorized-chats]
+  {}
+  (reduce-kv (fn [acc category chats]
+               (assoc acc category (add-on-press-handler-to-chats community-id chats)))
+             {}
+             categorized-chats))
+
+(defn community-header
+  [name]
+  [quo/text
+   {:accessibility-label :chat-name-text
+    :number-of-lines     1
+    :ellipsize-mode      :tail
+    :weight              :semi-bold
+    :size                :heading-1} name])
```

Comment:
I thought zprint would format this for us, but apparently not.

From the style guide, `name` should be on a separate line.

Ref:
https://github.com/bbatsov/clojure-style-guide#function-arguments-alignment